src/main/java/carRacing/ResultView.java

```diff
@@ -0,0 +1,15 @@
+package carRacing;
+
+import java.util.List;
+
+public class ResultView {
+
+	public static final String HYPHEN = "-";
+
```

Comment:
```suggestion
	private static final String HYPHEN = "-";
```
외부에 노출할 필요가 없는 상수이니 private 접근제한자로 변경하는 것이 좋을 것 같습니다.
그리고 HYPHEN 보다는 자동차 거리를 표현하는 다른 의미를 적어주는 것이 가독성에 도움을 줄 것 같아요!